face_authentication_lib/__init__.py,sha256=mH0TWSnmf2vyV5-62znyD3ytpIIXt51BH5Nl3Lxzojc,55
face_authentication_lib/exceptions.py,sha256=tDM8j0azPYnRTTCSRY7etZswcehhQHkELAGQAOXCu-o,872
face_authentication_lib/face_auth.py,sha256=EgzB3R4GEWiqOA6VmDxB58vCT_Z1TnDfmejNEW7DUCM,4893
face_authentication_lib-0.2.data/scripts/face_auth,sha256=61aqWVEBeBDNsj5Srvygox2BP2ajTXJR-xGpxa5hD1M,911
face_authentication_lib-0.2.dist-info/LICENSE.txt,sha256=UDHbX3jNT5sd2VndtYztx9gIPPtdyUt0O-raRCQVq6Q,1066
face_authentication_lib-0.2.dist-info/METADATA,sha256=bHGLR5uNVJtC5ftZS7UBnqWCI_ZwwMvvJHdri6aQFZ0,1375
face_authentication_lib-0.2.dist-info/WHEEL,sha256=G16H4A3IeoQmnOrYV4ueZGKSjhipXx8zc8nu9FGlvMA,92
face_authentication_lib-0.2.dist-info/top_level.txt,sha256=0g6Hq5DwKnbRYxXl3iiBo1A204wR7zobelJrUkFrgR0,24
face_authentication_lib-0.2.dist-info/RECORD,,
